handler.go

```diff
@@ -49,12 +51,19 @@ type item struct {
 	Comments []comment `json:"-"`
 }
 
+type User struct {
```

Comment:
Is there any particular reason we need to export the type?

Reply:
good catch. it does not need to be exported.

Reply:
fixed!